mypy_test.ini

```diff
@@ -6,3 +6,10 @@ exclude = (?x)(
     ^test/mypy_fails/*.*$
     | ^test/conftest.py$
   )
+
+[mypy-pymongo.synchronous.*,gridfs.synchronous.*]
+warn_unused_ignores = false
+disable_error_code = unused-coroutine
```

Comment:
Several places use `IS_SYNC` to call either the sync or async version of a network-level operation. This causes linting errors in the synchronous codebase due to the async coroutine for the `IS_SYNC = False` path missing an await.